images/bootstrap/Dockerfile

```diff
@@ -0,0 +1,20 @@
+# Copyright 2020 The KubeVirt Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+FROM gcr.io/k8s-testimages/bootstrap:v20201106-b01d6e7
+
+COPY [
+  "runner_wrapper.sh", \
```

Comment:
I see that we run `runner.sh` in pretty much all our prowjobs. Would it make sense to rename `runner.sh` and name the wrapper `runner.sh` instead?

Reply:
Good point, done PTAL